sdk/java-sdk-testkit/src/main/java/kalix/javasdk/testkit/KalixTestKit.java

```diff
@@ -111,6 +111,15 @@ public Settings withServiceName(final String serviceName) {
     public Settings withAclDisabled() {
       return new Settings(stopTimeout, serviceName, false);
     }
+
+    /**
+     * Enable ACL checking in this service.
+     *
+     * @return The updated settings.
+     */
+    public Settings withAclEnabled() {
+      return new Settings(stopTimeout, serviceName, true);
+    }
```

Comment:
Adding `withAclEnabled`, but keeping the `withAclDisabled` that is now less useful since it's the default. Shall we deprecate it?